integrations/tensorflow/bindings/python/iree/tf/support/CMakeLists.txt

```diff
@@ -24,7 +24,37 @@ iree_py_library(
     "trace_utils.py"
 )
 
-# TODO: Add tests.
+# Make the base dir that run_test.sh expects to exist. This dir doesn't exist
+# by default because we symlink these python files into bindings/python/iree.
+file(MAKE_DIRECTORY "${CMAKE_BINARY_DIR}/integrations/tensorflow/bindings/python/iree/tf/support")
```

Comment:
So note that run_test.sh isn't actually the thing that decides the name for this. That's actually https://github.com/google/iree/blob/9f0148fa15692bd16d9193fa3fc94f114114cad9/build_tools/cmake/iree_installed_test.cmake#L64 (and our other cmake test functions. Why have the logic in only one place!?)

Could you add a TODO here linking to https://github.com/google/iree/issues/5147?